plugins/diagnose.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: diagnose
+spec:
+  version: v0.1.1
+  homepage: https://github.com/alecjacobs5401/kubectl-diagnose
+  shortDescription: Find and debug Pods that are "Not Ready"
+  description: |
+    This plugin shows finds and displays debugging information for Pods
+    that are "Not Ready" in the current namespace.
+    Pods are deemed as "Not Ready" if they have any failing Pod Conditions
+    or have any containers that do not have a "Ready" value.
+    In addition, you can filter which pods you want to show based on labels
+    or field selectors (as well as by pod name(s)).
+  caveats: |
+    * The -A/--all-namespaces flag is currently not supported
```

Comment:
Please use `k8s.io/cli-runtime` to talk to the cluster. This will take care of all the standard flags.

Reply:
If possible, I'd like to push this to a later release version. The `podevents` plugin that was already approved and merged had the same caveat, so I'd like to take the time to update both to adhere to better standards, but still get this `sick-pods` in.